cmd/apexctl/device.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/google/uuid"
+	"github.com/redhat-et/apex/internal/client"
+	"github.com/redhat-et/apex/internal/models"
+	"github.com/urfave/cli/v2"
+)
+
+func listAllDevices(cCtx *cli.Context) error {
```

Comment:
nit: I would recommend following this type of format.
E.g 
```go

listAllDevicesCommand := &cli.Command{
    // ... flags and stuff ...
    Action: func(cCtx cli.Context) error {
        // Extract all the flags
        some := ctx.String("some")
        args := ctx.String("args")
        return listAllDevices(some, args)
    }
}

func listAllDevices(some, args string) error {
  // main function goes here
}
```

I'm certainly no urfave/cli expert, but this seems the only pattern I could find that makes these functions testable if you want.

```go
func TestListAllDevices() {
    // we can inject some data here for testing without messing around with os.Args and executing the subcommand etc...
    testListAllDevices("foo", "bar")
}
```